src/Assetic/Filter/LessFilter.php

```diff
@@ -0,0 +1,208 @@
+<?php namespace October\Rain\Assetic\Filter;
```

Comment:
We don't want this filter, just the LessphpFilter

Reply:
Gone! 
Any idea about the Code Quality error n LessphpFilter.php ?

Reply:
No idea.